### PR TITLE
Fixed a typo, added more info about filtering.md

### PR DESCRIPTION
**Pull request by Expertium**

### CODE REVIEW

**Comment by L-M-Sherlock** [comment]
Ready for merge? What about asking @user1823 to review it?

**Comment by Expertium** [comment]
> What about asking @user1823 to review it?

I don't think that's necessary here

**Comment by L-M-Sherlock** [comment]
OK. I will review it later.

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ For further information regarding the FSRS algorithm, please refer to the follow
 
 Total number of users: 19,990.
 
-Total number of reviews for evaluation: 707,964,360. Same-day reviews are excluded; only one review per day (the first chronologically) is used by each algorithm, except for FSRS-5. This is why the number of reviews is significantly lower than the 1.7 billion figure mentioned earlier. Plus, an outlier filter is also used.
+Total number of reviews for evaluation: 707,964,360. Same-day reviews are excluded; only one review per day (the first chronologically) is used by each algorithm, except for FSRS-5. This is why the number of reviews is significantly lower than the 1.7 billion figure mentioned earlier. Additionally, some reviews are filtered out, such as when the user manually changed the due date (which would count as a review), or when the user used what's called a "filtered deck" if "Reschedule cards based on my answers in this deck" was disabled. Finally, an outlier filter is applied.
```
Comment (user1823) [inline README.md]:
```suggestion
Total number of reviews for evaluation: 707,964,360.
Same-day reviews are excluded except in FSRS-5, i.e., each algorithm uses only one review per day (the first, chronologically). Some reviews are filtered out, for example, the revlog entries created on changing the due date manually or reviewing the cards in a filtered deck with "Reschedule cards based on my answers in this deck" disabled. Finally, an outlier filter is applied. These are the reasons why the number of reviews used for evaluation is significantly lower than the figure of 1.7 billion mentioned earlier. 
```

**Comment by user1823** [comment]
> What about asking @user1823 to review it?

When you don't really mean to ping, you should not use the @ sign. 😅

**Comment by L-M-Sherlock** [comment]
Sorry for annoying you. Actually I want to ask you.

**Review by L-M-Sherlock — APPROVED** [review]
LGTM. Thank you for your contributions.

**Comment by user1823** [comment]
> Sorry for annoying you. Actually I want to ask you.

No worries. But in that case, saying something like "@user1823, can you please review this?" is more clear than saying "What about asking @user1823 to review it?"

**Pull request merged**